docs/terms_and_conditions.md

```diff
@@ -0,0 +1,148 @@
+
+**V. PRAWA I OBOWIĄZKI USŁUGODAWCY, UŻYTKOWNIKA I UŻYTKOWNIKA NIEZALOGOWANEGO**
+1. Usługodawca zobowiązuje się do stałego i nieprzerwanego świadczenia Usług.
+2. Usługodawca zastrzega sobie prawo do:
+a. przejściowego zaprzestania świadczenia Usług ze względu na czynności konserwacyjne lub związane z modyfikacją Serwisu,
+b. wysyłania komunikatów technicznych, prawnych i transakcyjnych związanych z funkcjonowaniem Usług, w sposób wskazany w Załączniku nr 1 do Regulaminu, stanowiącym Wykaz i opis Usług,
+c. odmowy świadczenia Usług, jeżeli Użytkownik lub Użytkownik niezalogowany narusza Regulamin,
+d. dowolnej modyfikacji świadczonych Usług, narzędzi oraz sposobu działania Serwisu, poprzez zmianę Regulaminu.
+3. Usługodawca zastrzega sobie prawo do zaprzestania świadczenia Usług, usunięcia wszelkich danych Użytkownika lub Użytkownika niezalogowanego oraz podjęcia wszelkich innych dozwolonych prawem czynności związanych z Serwisami, z tytułu których Użytkownikowi lub Użytkownikowi niezalogowanemu nie będą przysługiwać żadne roszczenia wobec Usługodawcy.
+4. Użytkownik ma możliwość zarządzania Usługami za pośrednictwem Konta, w tym Ustawień (Prywatność) i zmiany podanych przez siebie danych w dowolnym momencie.
+5. W przypadku wystąpienia awarii lub innych zakłóceń w funkcjonowaniu systemów Usługodawcy, uniemożliwiających korzystanie z określonych Usług, funkcjonalności lub ustawień, Usługodawca podejmuje niezwłocznie wszelkie działania zmierzające do naprawienia takiej awarii lub zakłócenia.
+6. W przypadku skorzystania przez Użytkownika z Usług związanych z publicznym udostępnieniem w Serwisach swojego Wizerunku, na podstawie art. 81 Ustawy o prawie autorskim i prawach pokrewnych, Użytkownik wyraża zgodę na jego nieodpłatne utrwalanie, zwielokrotnienie i rozpowszechnienie przez Usługodawcę. Usługodawca oświadcza, że Wizerunek będzie wykorzystywany w celu poprawnej realizacji Usług, lub wykorzystywany będzie w celu identyfikacji Użytkownika, lub w celach promocyjnych, reklamowych i marketingowych Usługodawcy.
+7. Użytkownik oświadcza, że przysługują mu autorskie prawa majątkowe do Utworów. Użytkownik udziela Usługodawcy nieodpłatnej, niewyłącznej oraz nieograniczonej terytorialnie licencji na czas realizacji Usług, zezwalającej na korzystanie z Utworów na następujących polach eksploatacji, obejmujących w szczególności: wytwarzanie, zwielokrotnianie, publiczne odtworzenie i wyświetlanie, wprowadzenie do pamięci komputera i serwerów sieci komputerowych, umieszczanie w sieci Internet.
```

Comment:
To samo co wyżej. Jeśli nie definiujemy "Utworu", a jedynie wskazujemy, że rozumiemy to słowo zgodnie z zapisami jakiejś ustawy to używałbym małych liter.